browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -629,8 +631,19 @@ void RewardsDOMHandler::OnAutoContributePropsReady(
       props->reconcile_stamp,
       props->contribution_non_verified,
       props->contribution_min_visits,
+      false,
       base::Bind(&RewardsDOMHandler::OnContentSiteList,
                  weak_factory_.GetWeakPtr()));
+  rewards_service_->GetContentSiteList(
```

Comment:
this shouldn't happen here as now we are getting exclude list for other cases as well

Reply:
please create separate function for exclude

Reply:
done